proposals/0333-snapshot-include-block-id.md

```diff
@@ -0,0 +1,57 @@
+---
+simd: '0333'
+title: Serialize Block ID in Bank into Snapshot
+authors:
+  - Emily Wang
+category: Standard
+type: Core
+status: Review
+created: 2025-08-06
+feature:
+---
+
+## Summary
+
+Currently the block ID (the merkle root of the last FEC set) of the
+snapshot slot is not included in the snapshot Bank Fields. This would
+enforce the snapshot producer to include the block ID in the serialized
+bank.
+
+## Motivation
+
```

Comment:
Additionally it would be good to specify that in Alpenglow votes / certificates and related tracking is only performed on (Slot, block_id), so it's imperative that we have block_id available in snapshots.